api/types/src/convert.rs

```diff
@@ -179,8 +179,7 @@ impl<'a, S: StateView> MoveConverter<'a, S> {
         use aptos_types::transaction::Transaction::*;
         let aux_data = self
             .db
-            .get_transaction_auxiliary_data_by_version(data.version)
-            .ok();
+            .get_transaction_auxiliary_data_by_version(data.version)?;
```

Comment:
Is there a case where a transaction will never parse because of this?  Or I guess it looks like it wouldn't work before

Reply:
no as I know of